src/crypto-utils.ts

```diff
@@ -14,6 +14,17 @@ export function hexToNumber(hex: string): number {
   return parseInt(hex, 16)
 }
 
+export function hexAddrToUint8Array(hexAddr: string): Uint8Array {
```

Comment:
This is no longer necessary, lets remove it.

Reply:
Removed